api/fileProcessing/index.js

```diff
@@ -194,7 +194,8 @@ module.exports = function(app) {
    * @apiGroup FileProcessing
    *
    * @apiParam {JSON} data Data which defines the track (type specific).
-   * @apiParam {Number} algorithm Tracking algorithm version number.
+   * @apiParam {Number} AlgorithmId Database Id of the Tracking algorithm details retrieved from
+   * <a href="#api-fileprocessing-algorithm">Algorithm</a> request
```

Comment:
You can link to other sections like this: `(#Group:Name)`. So in this case `(#FileProcessing:Algorithm)`